docs/configuration_options.md

```diff
@@ -201,6 +201,7 @@
 | **<a href="#user-content-k8s_cert_manager_version" name="k8s_cert_manager_version">k8s_cert_manager_version</a>**  |  | N/A | SCT_K8S_CERT_MANAGER_VERSION
 | **<a href="#user-content-k8s_minio_storage_size" name="k8s_minio_storage_size">k8s_minio_storage_size</a>**  |  | 10Gi | SCT_K8S_MINIO_STORAGE_SIZE
 | **<a href="#user-content-k8s_log_api_calls" name="k8s_log_api_calls">k8s_log_api_calls</a>**  | Defines whether the K8S API server logging must be enabled and it's logs gathered. Be aware that it may be really huge set of data. | N/A | SCT_K8S_LOG_API_CALLS
+| **<a href="#user-content-k8s_serverless" name="k8s_serverless">k8s_serverless</a>**  | Defines whether the we enable the operator serverless options | N/A | SCT_K8S_SERVERLESS
```

Comment:
I'm not sure we want to make every single option of the operator to map to the SCT configuration, and it would be o.k. to bundle them all in "serverless" option

Reply:
for now, changed it